### DISCUSSION TITLE
"allowSkeletonHorseBreeding" / "allowZombieHorseBreeding"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#80] [Ideas]
Allows to breed the horses, but not just that, it allows to max the fixed Horse stats out from them. At least for Skeleton Horses, not sure if Zombie Horses having the same fixed values. From testing, jump values are different but every Skeleton Horse has the same speed value, which is not good.

**Comment by 1024-byteeeee** [comment]
- They are dead. 🤣
- 往 carpet 拓展中增加生物繁殖功能我认为是不合理的，幼年状态的模型依靠客户端进行渲染，因此除了鹦鹉之后我不会再制作关于生物繁殖的规则。
It is unreasonable to add biological reproduction functionality to the carpet expansion. Models in baby state rely on client-side rendering, so I will not create any rules regarding biological reproduction, except for parrots.

**Reply by ghost** [reply]
That's fine - then I need to come over with a different solution. Mods who tackle the horses directly.
